### PR TITLE
Feature/support comma separated version file paths

### PR DESCRIPTION
**Pull request by andresmatasuarez**
Addresses https://github.com/TriPSs/conventional-changelog-action/issues/43

@TriPSs please, review everything and let me know of any additional changes that need to be made before merging

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -96,3 +96,20 @@ jobs:
           github-token: ${{ secrets.github_token }}
           version-file: 'test-file.toml'
           version-path: 'package.version'
+
+  multiple-files:
```
Comment (TriPSs) [inline .github/workflows/test.yml:100]: Could you keep this inline with the other ones and call it `test-multiple-lines`

Comment (andresmatasuarez) [reply .github/workflows/test.yml:100]: you mean `test-multiple-files`, right? :)

Comment (andresmatasuarez) [reply .github/workflows/test.yml]: done f66f6a2

**Comment by TriPSs** [comment]
Pr looks good, thanks for your contribution!

**Pull request merged**